Wired.js

```diff
@@ -0,0 +1,300 @@
+{
+	"translatorID": "f054a3d9-d705-4d2e-a96a-258508bebba3",
+	"label": "Wired",
+	"creator": "czar",
+	"target": "^https?://(www\\.)?wired\\.(com|co\\.uk)",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-07-06 02:44:35"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2018 czar
+	http://en.wikipedia.org/wiki/User_talk:Czar
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+// attr()/text() v2 per https://github.com/zotero/translators/issues/1277
+function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
+
+
+function detectWeb(doc, url) {
+	if (/\/(\d{4}\/\d{2}|story|article)\//.test(url)) {
+		return "magazineArticle";
+	} else if (/\/(category|tag|topic)\/|search\/?\?q=|wired\.com\/?$|wired\.co\.uk\/?$/.test(url)) {
+		return "multiple";
+	} else return null;
+}
+
+
+function scrape(doc, url) {
+	var translator = Zotero.loadTranslator('web');
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48'); // embedded metadata
+	translator.setDocument(doc);
+	
+	translator.setHandler('itemDone', function (obj, item) {
+		item.itemType = "magazineArticle";
+		if (url.indexOf("wired.co.uk/article") != -1) {
+			item.publicationTitle = "Wired UK";
+			item.ISSN = "1357-0978";
+			item.date = Zotero.Utilities.strToISO(text(doc,'div.a-author__article-date')); // use LSON-LD when implemented in EM
+		} else { // if not wired.co.uk
+			item.publicationTitle = "Wired";
+			item.ISSN = "1059-1028";
+			item.date = attr(doc,'meta[name="DisplayDate"], meta[name="parsely-pub-date"]','content');
+			item.creators = [];
+			var authorMetadata = attr(doc,'meta[name="Author"], meta[name="parsely-author"]','content');
+			if (authorMetadata) {
+				item.creators.push(ZU.cleanAuthor(authorMetadata, "author"));
+			}
+			if (item.tags) { // catch volume/issue if in tags
+				var match = null;
+				for (var tagCount in item.tags) {
```

Comment:
don't use for ... in to iterate through an array. Either
`for (let i =0; i<item.tags.length; i++)` or `for (let tag of item. tags)` 
also note using `let` instead of `var` for variables only used in one instance locally.

(you may simply have gotten thrown by the new tag display; it's still an array, though)

Reply:
* converted to `let`
* `for ... of` produced a TypeError so reverted to `for ... in`